blog/2024.9.9-QF.md

```diff
@@ -9,6 +9,20 @@ tags: [公告]
 
 <!-- truncate -->
 
+:::info 竞猜
+
+随着 Qualifier 的到来，OFFC24 Cirno 的竞猜也已启动！所有人均可参加，竞猜正确数目第一者可获得**一个月的 osu!supporter**！（若有并列，则选取投稿时间前三名）
```

Comment:
Have we decided to change our tournament title to ˋOFFC24ˋ yet?

Reply:
Not yet? Don't know what they think on this.
Can change this to *OFFC 2024* if possible.

Reply:
On my side OFFC cirno is fine. We are not responsible for tournament name clash, in fact, it is not a big deal.